Sources/Workspace/Workspace.swift

```diff
@@ -110,80 +110,85 @@ public class Workspace {
     /// A struct representing all the current manifests (root + external) in a package graph.
     public struct DependencyManifests {
         /// The package graph root.
-        let root: PackageGraphRoot
+        internal let root: PackageGraphRoot
 
         /// The dependency manifests in the transitive closure of root manifest.
-        let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
+        internal let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
```

Comment:
We don't usually explicitly write internal, is there a reason behind this change?